README.md

```diff
@@ -101,12 +101,16 @@ On Red Hat-based operating systems, including Fedora, install gettext via
 
 1. In your `project.clj`, add `puppetlabs/i18n` to the `:dependencies` and to
    the `plugins`
-1. Run `lein i18n init`. This will
```

Comment:
Hmm, markdown should support automatically filling in list indices from `1.\n1.\n1.\n`.